app/models/ThangType.coffee

```diff
@@ -392,6 +392,12 @@ module.exports = class ThangType extends CocoModel
     props = _.without props, 'canCast', 'spellNames', 'spells'
     for stat, modifiers of itemConfig.stats ? {}
       stats[stat] = @formatStatDisplay stat, modifiers
+      if @heroStatModifier? and stats['maxHealth']? and !isNaN(stats['maxHealth'].display)
```

Comment:
Prefer `not` to `!` in CoffeeScript. Actually, this could be cleaner as:
```
      if @heroStatModifier? and stats['maxHealth']?.display > 0
```
We also should not do this if `@heroStatModifier?.health?.absolute` is 1.